src/compiler/resolutionCache.ts

```diff
@@ -1224,12 +1199,9 @@ export function createResolutionCache(resolutionHost: ResolutionCacheHost, rootD
 
     function canWatchTypeRootPath(nodeTypesDirectory: string) {

```

Comment:
Here the parameter is called `nodeTypesDirectory` - but this function feels like it expects any implicit or explicit set of type roots. Can you rename this as appropriate?